spec/factories/cards.rb

```diff
@@ -0,0 +1,6 @@
+ FactoryGirl.define do
+  factory :card do
+    original_text   "Door"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.